src/components/language/language.service.ts

```diff
@@ -537,8 +534,7 @@ export class LanguageService {
     session: ISession
   ): Promise<LanguageListOutput> {
     const label = 'Language';
-    // const baseNodeMetaProps = ['id', 'createdAt'];
-    // const unsecureProps = [''];
+
     const secureProps = [
```

Comment:
This has already been defined in `readOne`. Just move that to a private class prop or a top-level instead of duplicating it here.